samples/charts/scatter/index.html

```diff
@@ -0,0 +1,253 @@
+        </div>
+        <div class="panel-body">
+                      <pre>
+{
+    type: "scatter",
+        x : "Unemployment",
+        charts : [{type: "scatter",  y : "Area",color: "Inflation"}],
+        size : "GDP",
+        maxLength: 6,
+        width: 400,
+        height: 200,
+        "maxColor":"#ffff00",
+        "minColor":"#ff00ff"
+}
+                      </pre>
+
+            <table class="table table-bordered">
+                <thead>
+                <tr>
+                    <th>Property</th>
+                    <th>Value</th>
+                    <th>Description</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr class="info">
+                    <td>title</td>
+                    <td>String</td>
+                    <td>Title of the table</td>
+                </tr>
+                <tr class="danger">
+                    <td>x</td>
+                    <td>String</td>
+                    <td>cAttribute name for X-axis</td>
+                </tr>
+                <tr class="danger">
+                    <td>maxLength</td>
+                    <td>int</td>
+                    <td>Maximum length of chart datatable</td>
+
+                </tr>
+
+                <tr class="danger">
+                    <td>height</td>
+                    <td>px</td>
+                    <td>height of the chart</td>
+
+                </tr>
+
+                <tr class="danger">
+                    <td>charts</td>
+                    <td>object</td>
+                    <td>Collection of charts</td>
+
+                </tr>
+                </tbody>
+            </table>
+
+            <table class="table table-bordered">
+                <thead>
+                <tr>
+                    <th>Property</th>
+                    <th>Value</th>
+                    <th>Description</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr class="danger">
+                    <td>type</td>
+                    <td>String</td>
+                    <td>Title of the table</td>
+                </tr>
+                <tr class="danger">
+                    <td>y</td>
+                    <td>String</td>
+                    <td>Attribute name for Y-axis</td>
+
+                </tr>
+
+
+                </tbody>
+            </table>
+
+        </div>
+
+    </div>
+</div>
+
+<script>
+
+    var data =  [
+        {
+            "metadata" : {
+                "names" : ["Area","GDP","Inflation","Life.expect","Military","Pop.growth","Unemployment"],
+                "types" : ["linear","linear", "linear","linear","linear","linear","linear"]
+            },
+            "data": [
+
+                [43871,41600,3.5,79.91,0.8,0.03,4.2]
+
+
+                ]
+        }
+    ];
+
+
+    var config = {
+        type: "scatter",
+        x : "Unemployment",
+        charts : [{type: "scatter",  y : "Area",color: "Inflation"}],
+        size : "GDP",
```

Comment:
@dnwick Shall we have size attribute inside charts, as it is related to scatter chart